pkg/core/function.go

```diff
@@ -536,14 +538,11 @@ func (c *client) UpdateFunction(options UpdateFunctionOptions, log io.Writer) er
 		repoName := configuration.RevisionTemplate.Spec.Container.Image
 		publish := publishImage(repoName)
 
-		if buildImage == "" || runImage == "" {
```

Comment:
Is this error message replaced by https://github.com/projectriff/riff/pull/974/files#diff-962221b752f1f6c34ce677a7163a2875R150 ? This does not convey the same information to the user

Reply:
Yes. The two new messages were intended to be more precise and more specific. Since this is common code, we have to cover both create and update. The old message didn't make a lot of sense in the case of create. The new messages make sense in both cases.

Reply:
~~Well, my point is, I'm not sure it does (I'm talking about the update case). Previously, it hinted to the user that she was trying to rebuild locally ...~~

Hmm, scratch that. I now realize this was covering the case trying to build locally an "invoker" based function. That case is gone now anyways